src/slurm_plugin/clustermgtd.py

```diff
@@ -798,8 +798,6 @@ def _handle_powering_down_nodes(self, slurm_nodes, private_ip_to_instance_map):
                 powering_down_nodes.append(node)
 
         if powering_down_nodes:
-            log.info("Resetting powering down nodes: %s", print_with_count(powering_down_nodes))
-            reset_nodes(nodes=[node.name for node in powering_down_nodes])
```

Comment:
when is this resetting happening? as soon as the powering down is initiated? If not are there some downstream consequences if we don't reset it here and Slurm hasn't reset this value yet?

Reply:
let's also update the docstring by mentioning that we don't reset the nodeaddr since Slurm takes care of it

Reply:
Docstring updated. Made modification to code to account for the fact that nodeaddr are not reset for powering_down nodes.